src/frontend/src/scheduler/plan_fragmenter.rs

```diff
@@ -751,6 +760,8 @@ impl BatchPlanFragmenter {
                     lookup_join_parallelism
                 } else if source_info.is_some() {
                     0
+                } else if let Some(num) = self.batch_parallelism {
+                    min(num as usize, self.worker_node_manager.worker_node_count())
```

Comment:
So the term `parallelism` in batch scenario is for the physical worker node, instead of logical CPU core? This is different from streaming... cc @liurenjie1024

Reply:
I guess in this case the parallelism is not the physical worker node? Because in this case the parallelism can bigger than the physical worker node count. 
https://github.com/risingwavelabs/risingwave/blob/0a044419976c38e54c5e07f24f05a88597830fe7/src/frontend/src/scheduler/plan_fragmenter.rs#L751

And  this parallelism will not be used in schedule process.
https://github.com/risingwavelabs/risingwave/blob/0a044419976c38e54c5e07f24f05a88597830fe7/src/frontend/src/scheduler/distributed/stage.rs#L334